README.md

```diff
@@ -9,7 +9,23 @@
 <li>using TDD</li></ul>
 
 ## Development progress
-<p>We are working to develop functioning Stage 1 on 21.12.2020</p>
+<p>We are working to develop functioning Stage 2 on 28.12.2020</p>
+
+## Workflow
+- We use separate branches to develop each stage of project.
+- Each stage consist of at least three steps: documentation, tests and implementation. They are developed in this order and depend on each another.
```

Comment:
It should be "each other" last two words 
Please chceck

Reply:
You are right, I will correct it soon.